File: Chapter 2 List & Array/Array/Q80 Remove Duplicates from Sorted Array II/Solution.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        MAX_ALLOWED = 2
        result = 1
        times = 1
        for i in range(1, len(nums)):
            if nums[i] != nums[i - 1]:
                nums[result] = nums[i]
                result += 1
                times = 1
            elif times < MAX_ALLOWED:
                nums[result] = nums[i]
                result += 1
                times += 1
        return result

File: Chapter 2 List & Array/Array/Q80 Remove Duplicates from Sorted Array II/Solution.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums.length == 0){
            return 0;
        }
        int result = 1;
        int times = 1;
        for(int i = 1; i < nums.length; i++){
            if(nums[i] != nums[i - 1]){
                nums[result++] = nums[i];
                times = 1;
            }else if(times == 1){{
                    nums[result++] = nums[i];
                    times++;
                }
            }
        }
        return result;
    }
}